wdl_test/FileMetadata.wdl

```diff
@@ -0,0 +1,53 @@
+version 1.0
+
+task CheckFileUpdatedDateGCP {
+
+    meta {
+        description: "Checks if file was updated within a specified time (default: 1 day)"
+        note: "Specific to GCP"
+    }
+
+
+    input {
+        Array[String] file_paths
+        Int days_back = 1
+        String image_to_use
```

Comment:
Let's use tag instead of image name, for security reasons.

It looks like you just need `gsutil` available in the image?